packages/models/src/authorization/client.ts

```diff
@@ -59,51 +59,3 @@ export const Client = z.object({
 });
 
 export type Client = z.infer<typeof Client>;
-
-export const KeyWithClient = z.object({
-  JWKKey: z.object({
-    kty: z.string(),
-    keyOps: z.array(z.string()).optional(),
-    use: z.string().optional(),
-    alg: z.string().optional(),
-    kid: z.string(),
-    x5u: z.string().optional(),
-    x5t: z.string().optional(),
-    x5tS256: z.string().optional(),
-    x5c: z.array(z.string()).optional(),
-    crv: z.string().optional(),
-    x: z.string().optional(),
-    y: z.string().optional(),
-    d: z.string().optional(),
-    k: z.string().optional(),
-    n: z.string().optional(),
-    e: z.string().optional(),
-    p: z.string().optional(),
-    q: z.string().optional(),
-    dp: z.string().optional(),
-    dq: z.string().optional(),
-    qi: z.string().optional(),
-    oth: z
-      .array(
-        z.object({
-          r: z.string(),
-          d: z.string(),
-          t: z.string(),
-        })
-      )
-      .optional(),
-  }),
-  client: z.object({
-    id: ClientId,
-    consumerId: TenantId,
-    name: z.string(),
-    purposes: z.array(PurposeId),
-    description: z.string().optional(),
-    users: z.array(UserId),
-    kind: ClientKind,
-    createdAt: z.coerce.date(),
-    keys: z.array(Key),
-  }),
-});
```

Comment:
t's no useful to declare this type which is not persisted, and it's only used as an Api result. By the way, the embedded [client](https://github.com/pagopa/interop-be-monorepo/pull/758/files#diff-a764f99cffde894c5ed8bc42d54e7b5b06d44750711eeff58845ad1f93a1eea7L96-L106) should have been a `Client` object, and not re-declared from scratch.